### PR TITLE
EZP-28397: Update icons for table toolbar in AlloyEditor

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28397
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Screenshot:
table-icons

#### Checklist:
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+export default class EzBtnTableCell extends AlloyEditor.ButtonTableCell {
+    static get key() {
+        return 'eztablecell';
+    }
+
+    /**
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js]: I think we can skip the docs related to lifecycle methods in React components.

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    render() {
+        let buttonCommandsList;
+        let buttonCommandsListId;
+
+        if (this.props.expanded) {
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js]: I would move it to a separate method to make `render` function body as small as possible.

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecolumn.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+     * Lifecycle. Renders the UI of the button.
+     *
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
+        let buttonCommandsList;
+        let buttonCommandsListId;
+
+        if (this.props.expanded) {
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecolumn.js]: Same as above. The similar suggestion applies to all the situations similar to this one.

File: src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js (hunk 2 above)
Comment (dew326) [reply src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js]: Most of the `render` function is copied from AlloyEditor, I would like to keep it like they do for easier maintenance.

Comment (sunpietro) [reply src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-tablecell.js]: Ok, let's keep it then.

**Pull request merged**